easybuild/easyblocks/s/suitesparse.py

```diff
@@ -122,7 +124,7 @@ def configure_step(self):
         if cfgvars:
             try:
                 f = open(fp, "a")
-                f.write("# lines below added automatically by EasyBuild")
+                f.write("# lines below added automatically by EasyBuild\n")
```

Comment:
oh crap, this is pretty bad :)

one of the entries in `cfgvars` was being ignored, and to top it off, it's a random one (ordering of `cfgvars.items()` list is random)